src/App.js

```diff
@@ -1,25 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] [OPTIONAL] It would be nice if you implement your function to also `Class` base and import the `react` component 😃 

```javascript
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```